src/code/z_elf_message.c

```diff
@@ -1,60 +1,68 @@
 #include "global.h"
-#include "z64elf_message.h"
-
-ElfMessage sChildSariaMsgs[] = {
-    ELF_MSG_STRENGTH_UPG(SKIP, 3, false, 0),
-    ELF_MSG_FLAG(CHECK, 0x61, false, EVENTCHKINF_37),
-    ELF_MSG_END(0x64),
-    ELF_MSG_FLAG(CHECK, 0x62, false, EVENTCHKINF_25),
-    ELF_MSG_FLAG(CHECK, 0x63, false, EVENTCHKINF_37),
-    ELF_MSG_FLAG(CHECK, 0x65, false, EVENTCHKINF_43),
-    ELF_MSG_MEDALLION(CHECK, 0x66, false, ITEM_MEDALLION_FOREST),
-    ELF_MSG_MEDALLION(CHECK, 0x66, false, ITEM_MEDALLION_FIRE),
-    ELF_MSG_MEDALLION(CHECK, 0x66, false, ITEM_MEDALLION_WATER),
-    ELF_MSG_SONG(CHECK, 0x67, false, ITEM_SONG_STORMS),
-    ELF_MSG_MEDALLION(CHECK, 0x68, false, ITEM_MEDALLION_SPIRIT),
-    ELF_MSG_MEDALLION(CHECK, 0x68, false, ITEM_MEDALLION_SHADOW),
-    ELF_MSG_END(0x69),
+#include "z64hint_commands.h"
+
+HintCmd sChildSariaHints[] = {
+    HINT_CMD_STRENGTH_UPG(SKIP, 0, false, 3),
```

Comment:
```suggestion
    HINT_CMD_STRENGTH_UPG(SKIP, 0, false, 0x03),
```
(textId)

Reply:
this is number of entries to skip, like a branch. its confusing cause it usually is a text id unless the command type is SKIP lol

Reply:
bruh got me

Could you explain that on `HINT_CMD_TYPE_SKIP` ?
like "Uses textId as the amount of commands to skip (including this one)"

Reply:
I can, yeah
This type of thing is why I wanted to make the cmd type be the actual macro, so SKIP could get its own arg name and stuff, but I could not figure out how to make it work